tar_scm.py

```diff
@@ -885,6 +904,7 @@ def parse_args():
         args.submodules = True
     else:
         args.submodules = False
+    args.sslverify = True if args.sslverify == 'enable' else False
```

Comment:
I does not matter in this case because argparse already checks against choices and apply the default value if not specified. If something different than choices has been specified argparse will exit with return code 2.